lib/adapters/http/index.js

```diff
@@ -950,7 +950,7 @@ function HttpPouch(opts, callback) {
             if (returnDocs) {
               results.results.push(c);
             }
-            utils.call(opts.onChange, c);
+            opts.onChange(c);
```

Comment:
Heh taking a look at the code (https://github.com/pouchdb/pouchdb/blob/master/lib/deps/call.js) Im gonna assume it has ended up not being made redundant and we have just missed it and forgot to take it out